### DISCUSSION TITLE
Macintosh Plus and bluescsi

### DISCUSSION THREAD

**Discussion by Wawavoun** [erichelgeson/BlueSCSI-v1#83] [Q&A]
Hello,

I try to boot my Mac Plus from scsi with a bluescsi system.

I have an image I made myself :

create an empty hda file of 20 MB with dd
run minivmac and boot it with 7.0.1 install 1 floppy image
format the empty hda file
install the system (all hardwares and all printers) to this disk
for test reboot minivmac with this file, ok !
move this image to the sd board and rename it HD10_512_blabla.hda
connect the blusscsi to the scsi port (modified with a diode so dont need power).
start the Plus..

No boot and if I boot with a floppy the scsi disk is not visible...

The Mac Plus is a revision C with 1 MB on board.

Any advices or solution ? Should I suspect the scsi port of the motherboard ?

Thanks and regards.
Philippe

**Comment by Wawavoun** [comment]
I forgot
The bluescsi, the sd card and this image works well on a SE/30

**Comment by erichelgeson** [comment]
Any image created with mini vmac will not work with a BlueSCSI. Please use a Preformatted blank and Basalisk II emulator or a pre-made image to test.

**Comment by Wawavoun** [comment]
Thanks Erich.
You are right, premade image works and the Plus boot well !
I noticed that only one image should be active on the sd media or the Plus fail to boot and become crazy... 
Is that normal ?
Regards.
Philippe

**Reply by erichelgeson** [reply]
This usually happens when you are using different scsi drivers on each image. Multiple images should work fine.

**Reply by Wawavoun** [reply]
Hello,

Okay now I have my bluescsi working on the Mac Plus with 7.0.1 FR. This is working only if I put only one active image on the sd board.

Now I try to get two images working at the same time.

I use dd to create an empty file with a suitable name on the side of the image with the system disk.
Then the Plus boot well. But nothing about the second disk...
Scsiprobe can see it (same model and manufacturer as the system disk) and he look to be mounted (first cell black on his line)  but nothing on the desktop and no ask for formatting (which is what I expect after the boot)...

I have also try with a simple copy of the system disk(of course with another scsi rank). What I dont like in this case is that both disk have the same identifier.
In this case the Plus wont boot  (some noises and black screen)...

Any help appreciated.

Thanks.
Philippe

**Reply by erichelgeson** [reply]
> In this case the Plus wont boot (some noises and black screen)...

This is likely due to having the incorrect scsi driver (LIDO) which is not compatible with the Plus - the `dd`'d blank will need to be formatted - you will have to use the patched Apple SC HD util to do so. If you could grab the pre-formatted images with Apple drivers from the Usage page and give that a try you can rule that out.

I'm not sure what you mean by both disks have the same identifier - you can always change the ID or name of disks.

**Reply by Wawavoun** [reply]
But If I use a copy of the system disk (which boot so I suppose with the good driver) as second disk then for sure I will again have the good driver ? I dont understand...
Yes I speak about the disk names show into Macos not of the scsi ID of course.

**Reply by erichelgeson** [reply]
> But If I use a copy of the system disk (which boot so I suppose with the good driver) as second disk then for sure I will again have the good driver ?

Yes I would think so.

**Reply by Wawavoun** [reply]
But even in this case the Plus refuse to boot !

**Reply by erichelgeson** [reply]
Is there any way you can zip up the images you are using and share them? I can take a look and see what drivers are embedded and test in my Plus.

**Reply by Wawavoun** [reply]
Hello
I send the image by direct reply into gmail. Please confirm reception.
Thanks for the help.
Philippe

**Reply by erichelgeson** [reply]
I do not believe you can email reply to a github notice and the file would be attached. Maybe just upload the file to google drive, box, dropbox, mega or the like and share the link here.

**Comment by Wawavoun** [comment]
Hi,
The zipped image is 10 MB, too big for gmail...
Can I upload it somewhere ?
Philippe

Le lun. 4 avr. 2022 à 21:57, Eric Helgeson ***@***.***> a
écrit :
